examples/config-api/tests/config_api_test.bal

```diff
@@ -1,44 +1,44 @@
-import ballerina/test;
-import ballerina/io;
-import ballerina/http;
-
-boolean serviceStarted = false;
-
-function startService() {
-    //serviceStarted = test:startServices("config-api");
-}
-
-// Execute this test as ballerina test config-api -e hello.keystore.password=@encrypted:{jFMAXsuMSiOCaxuDLuQjVXzMzZxQrten0652/j93Amw=}
-// then enter 12345 as the secret
-@test:Config {
-    before: "startService",
-    after: "stopService"
-}
-function testFunc() {
-    // Invoking the main function
-    http:Client httpEndpoint = new("https://localhost:8085", config = {
-        secureSocket:{
-            trustStore:{
-                path: "${ballerina.home}/bre/security/ballerinaTruststore.p12",
-                password: "ballerina"
-            }
-        }
-    });
-    // Chck whether the server is started
-    //test:assertTrue(serviceStarted, msg = "Unable to start the service");
-
-    string expectedValue = "Hello World!";
-
-    // Send a GET request to the specified endpoint
-    var response = httpEndpoint->get("/hello");
-    if (response is http:Response) {
-        var res = response.getTextPayload();
-        test:assertEquals(res, expectedValue);
-    } else {
-        test:assertFail(msg = "Failed to call the endpoint:");
-    }
-}
-
-function stopService() {
-    //test:stopServices("config-api");
-}
\ No newline at end of file
+//import ballerina/test;
```

Comment:
Any reason to comment the test out?

Reply:
needs the command ```ballerina run example.bal --b7a.config.file=/path/to/conf/file/custom-config-file-name.conf``` to run the tests. There is a discussion on this in the group.

Reply:
@aashikam can't we do something like this https://github.com/ballerina-platform/ballerina-distribution/blob/d1d1f0d29c468a25d425be65deb25e379a9b4e09/ballerina/build.gradle#L1292

Reply:
If it cant.. lets use @test:Config{enable:false}

Reply:
The test file was recently edited to not to read it from the conf file. It should be file now.